app/src/main/java/org/simple/clinic/feature/Feature.kt

```diff
@@ -36,5 +36,5 @@ enum class Feature(
   AddingHealthIDsFromEditPatient(false, "adding_health_ids_from_edit_patient"),
   NotifyAppUpdateAvailableV2(false, "appupdate_enabled_v2"),
   MonthlyDrugStockReportReminder(false, "monthly_drug_stock_report_reminders_v1"),
-  OverdueSections(false, "overdue_section_improvements_v0")
+  OverdueSections(true, "overdue_section_improvements_v0")
```

Comment:
Can we turn off this feature flag, please?

Reply:
oh shoot, sorry 😅